google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/wire/SubscriberBuilder.java

```diff
@@ -51,6 +51,18 @@ public static Builder newBuilder() {
     return new AutoValue_SubscriberBuilder.Builder().setContext(PubsubContext.of());
   }
 
+  public static void addDefaultMetadata(
+      PubsubContext context,
```

Comment:
instead, have this take:

PubsubContext, RoutingMetadata, Builder

where Builder is declared like in addDefaultMetadata.